examples/post_training_quantization/torch/save_load_mobilenet_v2/quantize.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+from typing import Tuple
+
+import torch
+
+import nncf
+from examples.post_training_quantization.torch.save_load_mobilenet_v2.common import QUANTIZED_CHECKPOINT_FILE_NAME
```

Comment:
```suggestion
from common import QUANTIZED_CHECKPOINT_FILE_NAME
```
import from root of repository works only if set PYTHON_PATH

Reply:
Fixed